### PR TITLE
Unmute recently failing CCQ tests

### PR DESCRIPTION
**Pull request by quux00**

### CODE REVIEW

**Review by alex-spies — APPROVED** [review]

File: muted-tests.yml [hunk 1]
```diff
@@ -338,17 +326,11 @@ tests:
...
-- class: org.elasticsearch.xpack.esql.action.CrossClustersQueryIT
-  method: testCCSExecutionOnSearchesWithLimit0
-  issue: https://github.com/elastic/elasticsearch/issues/115129
-- class: org.elasticsearch.xpack.esql.ccq.MultiClusterSpecIT
-  issue: https://github.com/elastic/elasticsearch/issues/115135
 - class: org.elasticsearch.xpack.test.rest.XPackRestIT
   method: test {p0=esql/60_usage/Basic ESQL usage output (telemetry)}
   issue: https://github.com/elastic/elasticsearch/issues/115231
+- class: org.elasticsearch.backwards.MixedClusterClientYamlTestSuiteIT
+  issue: https://github.com/elastic/elasticsearch/issues/115213
```
[the remark this answers is not in the document]
Comment (quux00) [reply muted-tests.yml:333]: Yes, it is not an ESQL test. It is a _search test.

File: muted-tests.yml [hunk 2]
```diff
@@ -158,12 +158,6 @@ tests:
 - class: org.elasticsearch.xpack.inference.rest.ServerSentEventsRestActionListenerTests
   method: testErrorMidStream
   issue: https://github.com/elastic/elasticsearch/issues/113179
-- class: org.elasticsearch.xpack.esql.qa.multi_node.EsqlSpecIT
-  method: test {categorize.Categorize SYNC}
-  issue: https://github.com/elastic/elasticsearch/issues/113054
-- class: org.elasticsearch.xpack.esql.qa.multi_node.EsqlSpecIT
-  method: test {categorize.Categorize ASYNC}
-  issue: https://github.com/elastic/elasticsearch/issues/113055
```
[the remark this answers is not in the document]
Comment (quux00) [reply muted-tests.yml:166]: OK, thanks.

**Comment by alex-spies** [comment]
Thanks @quux00 !

**Comment by alex-spies** [comment]
## 💚 All backports created successfully

| Status | Branch | Result |
|:------:|:------:|:------|
|✅|8.x|[](https://github.com/elastic/elasticsearch/pull/115302)|

### Questions ?
Please refer to the [Backport tool documentation](https://github.com/sqren/backport)

**Pull request merged**